slick/src/test/java/docs/javadsl/SlickTest.java

```diff
@@ -2,13 +2,16 @@
  * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
  */
 
-package akka.stream.alpakka.slick.javadsl;
+package docs.javadsl;
 
 import akka.Done;
 import akka.NotUsed;
 import akka.actor.ActorSystem;
 import akka.stream.ActorMaterializer;
 import akka.stream.Materializer;
+import akka.stream.alpakka.slick.javadsl.Slick;
+import akka.stream.alpakka.slick.javadsl.SlickRow;
+import akka.stream.alpakka.slick.javadsl.SlickSession;
```

Comment:
Awesome. This is exactly the result we wanted when moving examples to a separate package.